lib/bitstyles_phoenix/component/breadcrumbs.ex

```diff
@@ -0,0 +1,140 @@
+              Baz
+              <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" class="a-icon a-icon--m u-fg--gray-30 u-margin-xs-left" focusable="false" height="16" width="16">
+                <use xlink:href="#icon-caret-right">
+                </use>
+              </svg>
+            </li>
+          </ol>
+        </nav>
+        """
+    ''',
+    extra_html: """
+    <svg xmlns="http://www.w3.org/2000/svg" hidden aria-hidden="true">
+      <symbol id="icon-caret-right" viewBox="0 0 100 100">
+        <path d="M34.23,93.36a5.57,5.57,0,0,1-7.89-7.87L61.91,50.08,26.34,14.51a5.57,5.57,0,0,1,7.89-7.87L73.66,46.06a5.58,5.58,0,0,1,0,7.88Z" fill-rule="evenodd"/>
+      </symbol>
+    </svg>
+    """
+  )
+
+  story(
+    "Breadcrumbs with custom classes and propertires",
+    '''
+        iex> assigns = %{}
+        ...> render ~H"""
+        ...> <.ui_breadcrumbs class="u-fg--warning" data-foo="bar">
+        ...>   <:item class="u-fg--brand-2" data-baz="foo">Foo</:item>
+        ...>   <:item>Bar</:item>
+        ...>   <:item>Baz</:item>
+        ...> </.ui_breadcrumbs>
+        ...> """
+        """
+        <nav aria-label="breadcrumbs" data-foo="bar">
+          <ol class="u-h6 a-list-reset u-flex u-flex-wrap u-items-center u-fg--warning">
+            <li class="u-margin-xs-right u-flex u-items-center u-fg--brand-2" data-baz="foo">
+              Foo
+              <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" class="a-icon a-icon--m u-fg--gray-30 u-margin-xs-left" focusable="false" height="16" width="16">
+                <use xlink:href="#icon-caret-right">
+                </use>
+              </svg>
+            </li>
+            <li class="u-margin-xs-right u-flex u-items-center">
+              Bar
+              <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" class="a-icon a-icon--m u-fg--gray-30 u-margin-xs-left" focusable="false" height="16" width="16">
+                <use xlink:href="#icon-caret-right">
+                </use>
+              </svg>
+            </li>
+            <li class="u-margin-xs-right u-flex u-items-center">
+              Baz
+              <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" class="a-icon a-icon--m u-fg--gray-30 u-margin-xs-left" focusable="false" height="16" width="16">
+                <use xlink:href="#icon-caret-right">
+                </use>
+              </svg>
+            </li>
+          </ol>
+        </nav>
+        """
+    ''',
+    extra_html: """
+    <svg xmlns="http://www.w3.org/2000/svg" hidden aria-hidden="true">
+      <symbol id="icon-caret-right" viewBox="0 0 100 100">
+        <path d="M34.23,93.36a5.57,5.57,0,0,1-7.89-7.87L61.91,50.08,26.34,14.51a5.57,5.57,0,0,1,7.89-7.87L73.66,46.06a5.58,5.58,0,0,1,0,7.88Z" fill-rule="evenodd"/>
+      </symbol>
+    </svg>
+    """
+  )
+
+  def ui_breadcrumbs(assigns) do
+    extra = assigns_to_attributes(assigns, [:icon_file, :item, :class])
```

Comment:
why are `icon_file` and `item` part of `extra`?

Reply:
They are not. the second parameter of `assigns_to_attributes` excludes them from `extra`